src/styles/_article.scss

```diff
@@ -0,0 +1,61 @@
+#article {
+  header {
+    position: relative;
+    background: $finsec-royal-blue;
+  }
+  main {
+    h2,
+    h3,
+    h4,
+    p {
+      margin-top: 20px;
+    }
+    section {
+      align-items: left;
+      margin-bottom: 40px;
+      max-width: 1260px;
+      img {
+        max-width: 100%;
+        height: auto;
+      }
+      h4 {
+        color: $finsec-grey;
+        padding-left: 100px;
+        letter-spacing: 0.5px;
+        text-transform: uppercase;
+        a {
+          color: $finsec-grey;
+        }
+        a:hover {
+          color: grey;
+        }
+      }
+      @media (max-width: 900px) {
```

Comment:
Uhm best to inverse this.

```
h4 {
   @media (max-width: 900px) {
      padding-left: 40px;
   }
}
```

Personally I would never add HTML selectors within an `@media`. The media queries are always at the very end of my CSS hierarchies. People that don't tend to get in trouble eventually.